test/integration/auth/mongodb_oidc.test.ts

```diff
@@ -0,0 +1,17 @@
+import { expect } from 'chai';
+
+describe('MONGODB-OIDC', function () {
+  beforeEach(function () {
+    const MONGODB_URI = process.env.MONGODB_URI;
+    if (!MONGODB_URI || MONGODB_URI.indexOf('MONGODB-OIDC') === -1) {
```

Comment:
```suggestion
    if (!MONGODB_URI || !MONGODB_URI.includes('MONGODB-OIDC')) {
```

Just a nice to have improvement

Reply:
there's another indexOf usage below I didn't comment on

Reply:
Updated